### ISSUE TITLE
MySQL: `create table` does not work for `unique key`

### ISSUE THREAD

**Issue by kayman-mk** [sqlfluff/sqlfluff#2515]
## Expected Behaviour
SQLFluff should pass lint check when create unique constraint in MySQL. Please refer to https://dev.mysql.com/doc/refman/8.0/en/create-table.html

## Observed Behaviour
Linting fails with error message `Found unparsable section`.

## Steps to Reproduce
Prepare a file create_table.sql:

```sql
create table a(
  b int unique key
);
```

Run sqlfluff lint --dialect=mysql create_table.sql

Output will be
```
== [a.sql] FAIL
L:   1 | P:   1 |  PRS | Line 1, Position 1: Found unparsable section: 'create
                       | table a(\nb int unique key\n)'
```

## Dialect
Dialect is MySql.

## Version

0.9.3

## Configuration

No special configuration.
